lib/oban/peers/global.ex

```diff
@@ -69,7 +69,12 @@ defmodule Oban.Peers.Global do
 
   @impl GenServer
   def handle_info(:election, %State{} = state) do
-    locked? = :global.set_lock(key(state), nodes(), 0)
+    meta = %{conf: state.conf, peering_module: __MODULE__}
```

Comment:
Let's make the key `peer`, which matches the plugin spans:

```suggestion
    meta = %{conf: state.conf, leader: state.leader?, peer: __MODULE__}
```